### PR TITLE
docs: new search experience [ci skip]

### PR DESCRIPTION
**Pull request by kevinlul**
Preview: 
https://github.com/DawnbrandBots/bastion-bot/tree/docs-new-search#readme
https://github.com/DawnbrandBots/bastion-bot/blob/docs-new-search/docs/card-search.md
and proofread!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,9 +8,10 @@ _Yu-Gi-Oh! Trading Card Game_ and _Official Card Game_. This is the repository f
 to take advantage of new features offered by Discord and improve reliability.
 
 The new instance runs concurrently with the old instance.
-Features implemented here will be gradually rolled out to the live bot. For most documentation, please
-refer to the old [Bastion Classic](https://github.com/AlphaKretin/bastion-bot) repository. For
-Slash Command documentation, look in [`docs/commands`](/docs/commands).
+Features implemented here will be gradually rolled out to the live bot.
+New `<>` card search documentation is [HERE](/docs/card-search.md).
```
Comment (AlphaKretin) [inline README.md:12]: the allcaps is a bit out of place. Markdown provides a lot of more natural ways to draw attention. IDK whats proven to best get attention though

**Review by AlphaKretin — APPROVED** [review]
Didn't notice any typos going over it, though it's possible my brain autocorrected them. Happy with the conveyance of information here - the one complaint I have is subjective.

File: README.md (hunk 1 above)
Comment (kevinlul) [reply README.md:12]: Ah, I can bold it instead?

Comment (AlphaKretin) [reply README.md:12]: Yeah, that sounds good

**Pull request merged**